assets/nav.css

```diff
@@ -112,6 +112,11 @@ a.nav-link:hover, a.nav-link:focus {
     background-color: rgba(9, 114, 118, 0.08);
 }
 
+.modal-body img {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+}
 /*noinspection CssNonIntegerLengthInPixels*/
```

Comment:
u dont need css, use `d-block` and `mx-auto` instead.

Reply:
<img width="643" alt="Screenshot 2024-07-24 at 23 44 42" src="https://github.com/user-attachments/assets/2073839e-9af4-4486-a91e-ad838e30b1d6">
Hi Jared, for some reason when I use that the img doesn't centre

Reply:
@lynngoh7 

afk, a bit hard to debug...

try check devtools, do the classes apply to the element?